packages/docs/site/docs/developers/05-local-development/04-wp-playground-cli.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Playground CLI
+slug: /developers/local-development/wp-playground-cli
+---
+
+# Playground CLI
+
+[@wp-playground/cli](https://www.npmjs.com/package/@wp-playground/cli) is the new command-line tool designed to simplify the development and testing flow. It runs a WordPress instance locally with Playground, and with the CLI, it is possible to auto-mount an environment with a plugin, theme, or WordPress installation. But if you need flexibility, the CLI supports mounting commands to personalize your local environment.
+
+**Key features:**
+
+-   **Quick Setup**: Set up a local WordPress environment in seconds.
+-   **Flexibility**: Allows for configuration to adapt to different scenarios
+-   **Simple Environment**: No extra configuration, just a compatible Node installation, and you are ready to use it.
+
+## Requirements
+
+The Playground CLI requires Node.js 20.18 or higher, which is the recommended Long-Term Support (LTS) version. You can download it from the Node.js website.
+
+## Quickstart
+
+Running the Playground CLI is as simple as go to a command-line terminal and running:
+
+```bash
+npx @wp-playground/cli@latest server
+```
+
+The previous command, you only get a fresh WordPress, instance to test. Most of the developers want to see their work running, If this is your case, test a plugin or a theme. You can run the CLI on your project folder and run the Playground CLI with the `--auto-mount` flag:
+
+``bash
+cd my-plugin-or-theme-directory
+npx @wp-playground/cli@latest server --auto-mount
+
+````
+
+### Choosing a WordPress Version
+
+By default, the CLI loads the latest stable version of WordPress and PHP 8.0 due to its improved performance. To specify your preferred versions, you can use the flag `--wp=<version>` and `--php=<version>`:
+
+```bash
+npx @wp-playground/cli@latest server --wp=6.8 --php=8.4
+````
+
+### Mounting folders manually
+
+Some projects have a specific structure that requires a custom configuration, for example, your repo contains all the files from the `/wp-content/` folder. So this scenario you can specify to the Plaground CLI you will mount your project from that folder using the flag `--mount`.
+
+```bash
+npx @wp-playground/cli@latest server --mount=.:/wordpress/wp-content/plugins/
```

Comment:
Nice! Just noticing we'll need to coordinate updating this with merging https://github.com/WordPress/wordpress-playground/pull/2281 as that will break some CLI BC.